app/services/edge_cache/bust_podcast_episode.rb

```diff
@@ -0,0 +1,18 @@
+module EdgeCache
+  class BustPodcastEpisode < Bust
+    def self.call(podcast_episode, path, podcast_slug)
+      return unless podcast_episode && path && podcast_slug
+
+      podcast_episode.purge
+      podcast_episode.purge_all
+
+      begin
+        bust(path)
+        bust("/#{podcast_slug}")
+        bust("/pod")
+      rescue StandardError => e
+        Rails.logger.warn(e)
```

Comment:
I'm not sure why we rescue errors here. I'm guessing it's something with podcast episodes?